From 04e2c5053e765245c998902c06017bf32f6e11e1 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Mon, 18 Jun 2018 14:58:31 +0900
Subject: [PATCH 3971/5242] MLK-18625-3: dts: arm64: imx8mq dcss update ext
 phy clk src

commit  854c9d74a0bded4d1ebae9ebde37ffc813ed91f5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update dcss reference for external 27m differential
clock source

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Reviewed-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
(cherry picked from commit 63f49f8d694fe578f1a7e6dfc157e6af5cfc7ff5)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 642ebd2..f371222 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -558,7 +558,7 @@
 		assigned-clock-parents = <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 					 <&clk IMX8MQ_SYS1_PLL_800M>,
 					 <&clk IMX8MQ_SYS1_PLL_800M>,
-					 <&clk IMX8MQ_VIDEO2_PHY_27MHZ>;
+					 <&clk IMX8MQ_CLK_PHY_27MHZ>;
 		assigned-clock-rates = <594000000>,
 				       <800000000>,
 				       <400000000>,
-- 
1.7.9.5

